relay/pages/v/[orgName]/[repoName]/edit.tsx

```diff
@@ -35,6 +36,7 @@ export const EditProjectPage: React.FC = (): JSX.Element => {
     valist.getPendingRepoDevs(orgName, repoName).then(setPendingKeys),
     valist.getPendingRepoThresholds(orgName, repoName).then(setPendingThresholds),
     valist.getPendingReleases(orgName, repoName).then(setPendingReleases),
+    valist.getLatestRelease(orgName, repoName).catch(console.log).then(setLatestRelease),
```

Comment:
It seems like this could just use the helper function you made in this PR 😎

https://github.com/valist-io/valist/pull/86

Reply:
Meant to delete this PR, duplicate PR with a different branch!